app/gui/view/debug_scene/cloud-dashboard/src/projects_table.rs

```diff
@@ -1,25 +1,34 @@
-//! Module containing the [`ProjectsTable`], which is used to display a list of [`Project`]s in the
-//! Cloud dashboard.
+//! Module containing the [`ProjectsTable`] implementation, which is used to display a list of
+//! [`Project`]s in the Cloud dashboard.
 //!
 //! The list of [`Project`]s, as rendered, contains information about the state of the [`Project`],
 //! the data associated with the [`Project`], the permissions assigned to it, etc. Buttons for
 //! interacting with the [`Project`]s (e.g., starting and stopping the [`Project`]s) are also
 //! rendered.
 //!
+//! # Note
+//!
+//! The [`ProjectsTable`] *type* itself is declared in [`crate::projects_entry`], but all of the
+//! implementation is contained in this module. This is done to ensure a one-way dependency between
+//! this module and the [`crate::projects_entry`] module. That module also depends on the
+//! [`ProjectsTable`] type definition, but aside from that is self-contained.
+//!
 //! [`Project`]: ::enso_cloud_view::project::Project
+//! [`ProjectsTable`]: crate::projects_entry::ProjectsTable
```

Comment:
AFAIK the cyclic dependencies between _modules_ are usually not a problem in Rust, only the _crates_.

Reply:
You are correct, just seemed like cleaner organization at the time. But yeah in hindsight unnecessary. Will fix.